From f37903004d93e7d7b008b2667881159bd8fee1e1 Mon Sep 17 00:00:00 2001
From: Pranav Kant <pranavk@libreoffice.org>
Date: Sun, 6 Dec 2015 23:41:57 +0530
Subject: [PATCH] tdf#96246: Make pRenderingArguments nullable

Type gchar* is not nullable by convention.

See:
https://wiki.gnome.org/Projects/GObjectIntrospection/Annotations#Nullable_parameters

Change-Id: Ibfee816a3ef2d29c7376071fb61eda7bf0538efb
Reviewed-on: https://gerrit.libreoffice.org/20425
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Miklos Vajna <vmiklos@collabora.co.uk>
(cherry picked from commit 9e9703389e015d0f3520344715df6719559362e4)
---
 include/LibreOfficeKit/LibreOfficeKitGtk.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/LibreOfficeKit/LibreOfficeKitGtk.h b/include/LibreOfficeKit/LibreOfficeKitGtk.h
index c947ce3..8b6092c 100644
--- a/include/LibreOfficeKit/LibreOfficeKitGtk.h
+++ b/include/LibreOfficeKit/LibreOfficeKitGtk.h
@@ -65,7 +65,7 @@ GtkWidget*                     lok_doc_view_new_from_widget        (LOKDocView*
  * lok_doc_view_open_document:
  * @pDocView: The #LOKDocView instance
  * @pPath: (transfer full): The path of the document that #LOKDocView widget should try to open
- * @pRenderingArguments: lok::Document::initializeForRendering() arguments.
+ * @pRenderingArguments: (nullable): lok::Document::initializeForRendering() arguments.
  * @cancellable:
  * @callback:
  * @userdata:
-- 
2.5.0

